### PR TITLE
{lib}[foss/2021a] TensorFlow v2.4.4 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/7a46fb316bdbc34cfe44c8a006901c0c for a full test report.

**Review by branfosj — CHANGES_REQUESTED** [review]
Based on a comparison with `TensorFlow-2.4.1-foss-2020b.eb`.

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.4.4-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+        'modulename': 'absl',
+        'checksums': ['72d782fbeafba66ba3e525d46bccac949b9a174dbf66233e50ece09ee688dc81'],
+    }),
+    ('astunparse', '1.6.3', {
+        'checksums': ['5ad93a8456f0d084c3456d059fd9a92cce667963232cbf763eac3bc5b7940872'],
+    }),
+    ('grpcio', '1.43.0', {
+        'modulename': 'grpc',
+        'checksums': ['735d9a437c262ab039d02defddcb9f8f545d7009ae61c0114e19dda3843febe5'],
+    }),
```
[the remark this answers is not in the document]
Comment (lexming) [reply easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.4.4-foss-2021a.eb:101]: fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/14970/commits/ef56865131792fcac3688827424140fc78d3017e, thanks

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.4.4-foss-2021a.eb [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+]
+
+sanity_pip_check = True
+use_pip = True
+
+
+# Dependencies created and updated using findPythonDeps.sh:
+# https://gist.github.com/Flamefire/49426e502cd8983757bd01a08a10ae0d
+exts_list = [
+    ('h5py', '2.10.0', {
```
[the remark this answers is not in the document]
Comment (lexming) [reply easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.4.4-foss-2021a.eb:54]: fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/14970/commits/ef56865131792fcac3688827424140fc78d3017e, thanks

**Comment by lexming** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bber0501u03a.bb2.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/004415140ac8c568dfc7e0e1823af228 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 12 out of 12 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/513e4ed57c720a9a6a38638a0f9285ec for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/6b43f38bce7594d82f5b3996ce3ccf69 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bber0501u03a.bb2.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/1c35cd552659c26446629b0932ca3540 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @lexming!

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node316.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/92f3a290696d82515d81f055e69e7f65 for a full test report.

**Pull request merged**